packages/payment-processor/src/payment/index.ts

```
@@ -99,6 +101,8 @@ export async function payRequest(
     case ExtensionTypes.PAYMENT_NETWORK_ID.ERC20_FEE_PROXY_CONTRACT:
     case ExtensionTypes.PAYMENT_NETWORK_ID.ERC20_TRANSFERABLE_RECEIVABLE:
       return payErc20Request(request, signer, amount, undefined, overrides);
+    case ExtensionTypes.PAYMENT_NETWORK_ID.ERC20_HINKAL_WALLET:
+      return payErc20RequestHinkalWallet(request, signer, amount!);
```

Biome

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)